src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkBase.ControlType;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class ShooterSubsystem extends SubsystemBase {
+  /** Creates a new TRShooterSubsystem. */
+  private double shooterVelo;
+
+  private double TLP;
+  private double TLI;
+  private double TLD;
+
+  private double BRP;
+  private double BRI;
+  private double BRD;
+
+  private double feederVelo;
+
+  private CANSparkMax TLmotor1;
+  private CANSparkMax TLmotor2;
+  private CANSparkMax BRmotor1;
+  private CANSparkMax BRmotor2;
+  private CANSparkMax feederMotor;
+
+  public ShooterSubsystem() {
+    TLP = 1.0;
```

Comment:
1 is an aggressive P value for a sparkmax.  

Think you want to start with more like 1e-6

Reply:
Done.